marklogic-unit-test-client/src/main/java/com/marklogic/test/unit/JUnitTestCase.java

```diff
@@ -12,13 +12,19 @@ public class JUnitTestCase {
 	private String classname;
 	private String name;
 	private double time;
+  private int tests;
```

Comment:
Can you make sure your editor is using the .editorconfig file in this repository? We don't want any changes here due to different whitespace (of course these new lines are fine, but they should use the same formatting as existing lines).